jupyterlab_gitsync/README.md

```diff
@@ -0,0 +1,7 @@
+# JupyterLab Git Sync Extension
```

Comment:
How should I install this plugin? Can you give some quick instruction here?

Reply:
Development instructions are linked in the readme

Reply:
I think Yun is asking for usage instructions, which would be for people who simply want to use the plugin. Development instructions should be for people who want to contribute to or work on the plugin.

Reply:
Currently there's no easy way to run the extension outside a dev environment. I know most of the other projects go through a deploying process before they have a simple installation procedure (using npm or pip), so Yun said it's fine to just include the development instructions for now.

Reply:
sgtm